lib/package.json

```diff
@@ -40,11 +40,15 @@
     "react-dom": "^16.2.0"
   },
   "dependencies": {
+    "date-fns": "^1.29.0",
```

Comment:
What do you think about making `material-ui-pickers` to not rely on date libraries at all?
We can add `moment-utils` and `datefns-utils` as peerDependencies (each with `moment` or `dateutils` dependencies accordingly)

Reply:
Of course I will remove date-fns dependency and will let user to choose which utils to use moment or date-fns. And there would not peerDependecies at all